server/atl_requests/__init__.py

```diff
@@ -0,0 +1,58 @@
+from server.models import Authentication
+import atlassian_jwt
+import requests
+from urllib.parse import urlencode
+from server.atl_requests.lib import AtLib
+
+
+class Ctx:
+    def __init__(self, org):
+        self.tenant_info = Authentication.query.filter_by(baseUrl=org).first()
+        self.base_url = self.tenant_info.baseUrl
+        self.key = self.tenant_info.key
+        self.shared_secret = self.tenant_info.sharedSecret
+
+
+class AtlassianRequest:
+    def __init__(self, org):
+        self.ctx = Ctx(org)
```

Comment:
I think you'd be better suited passing the Context into the initializer rather than creating it. This will help when you start writing tests ;)